doc/source/whatsnew/v0.24.0.rst

```diff
@@ -648,6 +648,52 @@ that the dates have been converted to UTC
     pd.to_datetime(["2015-11-18 15:30:00+05:30",
                     "2015-11-18 16:30:00+06:30"], utc=True)
 
+
+.. _whatsnew_0240.api_breaking.read_csv_mixed_tz:
+
+Parsing mixed-timezones with :func:`read_csv`
```

Comment:
understand why u r putting this on 0.24.0 so does this mean u r going to rebuild docs and repush?

Reply:
I can if needed. The url off pandas.pydata.org points to `/stable`, so the whatsnew most people see for 0.24.0 will include this change.